src/server/object_services/map_deleter.js

```diff
@@ -43,6 +44,17 @@ async function delete_chunks_if_unreferenced(chunk_ids) {
     }
 }
 
+/**
+ * @param  {nb.ObjectMD} object
```

Comment:
Please add a comment:
```suggestion
 * Delete the object_md of objects that have no parts.
 * This is needed for the cleanup of cache buckets once all parts are evicted.
 * @param  {nb.ObjectMD} object
```